src/core/components/LinkifyMarkdown/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import sanitize from 'sanitize-html';
 
-const MARKDOWN_LINK_REGEX = /\[([^\\[]+)\]\((.*)\)/gm;
+const MARKDOWN_LINK_REGEX = /\[([^\\[]+)\]\(?(\S*)?\)/gm;
```

Comment:
![image](https://user-images.githubusercontent.com/1910749/207832918-7f60ea66-8c90-499d-b7c4-848d96272b13.png)

Reply:
Works fine with common cases, is not foolproof in some cases